### DISCUSSION TITLE
Multiwindow / Splitscreen

### DISCUSSION THREAD

**Discussion by MB175** [usebruno/bruno#574] [Ideas]
Right now I am testing some stuff where I need to execute two different request quite often at different times.

I think supporting multiple windows (like undocking tabs and have them floating) or split screen would be very handy

**Comment by MilenFrom** [comment]
I would've used but this is a deal breaker to me
I used many different requests compares and this limitation is very frustrating

**Comment by reynadev-oss** [comment]
This is a great suggestion! A multi-window or[ split-screen](https://smartwindows.app/blog/how-to-split-screen-in-windows-10-and-11/) feature would be incredibly useful for running concurrent requests—definitely a powerful enhancement for productivity.

**Comment by jpieres** [comment]
Supporting multiple windows would be a great improvement. Totally need it.

**Comment by eberteogam** [comment]
Any update on this. I'd like to see multiple tabs at the same time. Will v4.0 include this feature??
